src/app/(auth)/signup/page.tsx

```diff
@@ -76,14 +75,10 @@ export default function SignUp() {
 
   return (
     <>
-      {!emailSentCount && (
+      {emailSentCount === 0 && (
         <>
-          <SpacerDiv $gap={0.625}>
-            <AuthHeading>Sign Up</AuthHeading>
-            {errorMessage !== '' && (
-              <P $color={COLORS.redMid}>{errorMessage}</P>
-            )}
-          </SpacerDiv>
+          <AuthHeading>Sign Up</AuthHeading>
```

Comment:
as mentioned in the other comment, i think we can use H1 here instead of AuthHeading